napari/components/viewer_model.py

```diff
@@ -444,10 +444,6 @@ def _on_active_layer(self, event):
             self.camera.mouse_zoom = active_layer.mouse_zoom
             self._update_status_bar_from_cursor()
 
-    @staticmethod
-    def rounded_division(min_val, max_val, precision):
```

Comment:
I'm going to be that guy again, but this is public, so shouldn't be removed without a deprecation. If it was buried somewhere in a `utils` module, I wouldn't be so picky, but this can be accessed by something as short as `napari.Viewer.rounded_division`.

Reply:
Ugh... I mean, you're right, but this shouldn't have been here in the first place, and there's *much* bigger stuff breaking already together with this, so it feels like the value of holding this around is very little :/ It feels like we keep bloating our code with deprecations we then have to maintain/remove at some point, but without real benefits. But if you feel strongly about it, I'll readd it with deprecation :+1:

Reply:
I don't feel *too* strongly about removing this without a deprecation, so I'm happy to let you make the call here. Though might be good to check in with @jni who have the thumbs up to my comment above.

I agree this should have never been here in the first place and I do feel more strongly about bloating our public API in the first place. So one reason I like the deprecation is as a form of penance - a little suffering can help learning sometimes :D